backend/src/stats.js

```diff
@@ -93,59 +106,78 @@ async function aggregateNewAccountsCountByDate() {
   }
 }
 
-async function aggregateNewContractsCountByDate() {
+async function aggregateDeletedAccountsCountByDate() {
   try {
-    const newContractsByDate = await queryNewContractsCountAggregatedByDate();
-    NEW_CONTRACTS_COUNT_AGGREGATED_BY_DATE = newContractsByDate.map(
-      ({ date: dateString, new_contracts_count_by_date }) => ({
+    const deletedAccountsCountAggregatedByDate = await queryDeletedAccountsCountAggregatedByDate();
+    DELETED_ACCOUNTS_COUNT_AGGREGATED_BY_DATE = deletedAccountsCountAggregatedByDate.map(
+      ({ date: dateString, deleted_accounts_count_by_date }) => ({
         date: formatDate(new Date(dateString)),
-        contractsCount: new_contracts_count_by_date,
+        accountsCount: deleted_accounts_count_by_date,
       })
     );
-    console.log("NEW_CONTRACTS_COUNT_AGGREGATED_BY_DATE updated.");
+    console.log("DELETED_ACCOUNTS_COUNT_AGGREGATED_BY_DATE updated.");
   } catch (error) {
     console.log(error);
   }
 }
 
-async function aggregateUniqueDeployedContractsCountByDate() {
+async function aggregateLiveAccountsCountByDate() {
```

Comment:
this is the most important part for this PR and check carefully for this function